modules/ROOT/pages/guided-setup.adoc

```diff
@@ -66,20 +66,24 @@ Otherwise, provide these app details:
 
 == Configure Users
 
-Give your user the *Exchange Contributor* role:
+Give your user the Exchange contributor permission:
 
 . In your Anypoint Platform organization, click *Access Management* > *Users*.
 . Select your user.
-. Click *Role* > *Add role by name*.
-. Add the role *Exchange Contributor*.
+. Click *Permissions* > *Add permissions*.
+. Ensure that your user has the permission *Exchange Contributor*.
 
-Map community user to Anypoint role:
+Give community users the Exchange viewer permission:
 
 . Ensure that you have Exchange Administrator permissions.
-. In your Anypoint Platform organization, navigate to *Access Management* and click *Roles*.
-. In *Exchange Viewers*, set a new external group called `Community User` and save it.
+. In your Anypoint Platform organization, navigate to *Access Management* and click *Teams*.
+. Create a new team named *Community Users*.
+. Click *Settings*.
+. Select the parent team.
```

Comment:
Maybe we should be explicit here about which team is the parent?
Is it your new team? Some existing team? or just "the larger team to which you are adding your new team" (or something similar)?

Reply:
True! Fixed.